akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcProjectionSpec.scala

```diff
@@ -921,8 +922,13 @@ class JdbcProjectionSpec
         concatStr.text shouldBe "abc|def|ghi"
       }
       withClue(s"check: last seen offset is 2L") {
-        val offset = offsetStore.readOffset[Long](projectionId).futureValue.value
-        offset shouldBe 2L
+        val configForFutureValue = implicitly[PatienceConfig]
+        val eventuallyTimeout = Timeout(configForFutureValue.timeout.scaledBy(3))
+        eventually(eventuallyTimeout) {
+          val offset = offsetStore.readOffset[Long](projectionId).futureValue.value
+          offset shouldBe 2L
+        }
```

Comment:
This is overly complex. A simple `eventually{...}` wrap (without all the specific timeout math) will be fine.